packages/tracing/src/browser/browsertracing.ts

```diff
@@ -148,6 +148,7 @@ export class BrowserTracing implements Integration {
 
     const { _metricOptions } = this.options;
     startTrackingWebVitals(_metricOptions && _metricOptions._reportAllChanges);
+    startTrackingLongTasks(_metricOptions && _metricOptions._reportLongTasks);
```

Comment:
IMO we `rm -rf` the option and just :shipit:

Reply:
yeah I'm fine with that